app/models/invited_user.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class InvitedUser < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :invite
```

Comment:
We should add active record validations for `invite_id`. Mainly presence and uniqueness.

Reply:
I've added validation for presence of `invite_id` but not sure if we should check for uniqueness because there will be multiple records for invite redemptions for multiple use invites.

Reply:
Oops I meant adding a uniqueness validation for invited id scoped to the user_id where user _id is not null. The logic should be the same as the uniqueness index which was added.

Reply:
Understood, done.